api/api.go

```diff
@@ -15,7 +15,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/hashicorp/go-cleanhttp"
+	cleanhttp "github.com/hashicorp/go-cleanhttp"
```

Comment:
There are a bunch of import changes like this, was it intentional? 

If this is goimports doing it might be best to document what version the whole team should be using in the project README so we can all be consistent.

Reply:
It was intentional. It is the output from running `make check`. I just fixed all the gofmt and goimports changes from go 1.10 -> 1.11